src/richie/apps/courses/models/category.py

```diff
@@ -83,12 +83,22 @@ class CategoryPluginModel(PagePluginMixin, CMSPlugin):
     """
     Category plugin model handles the relation between CategoryPlugin
     and their Category instance
+
+    We only propose to link category objects:
+    - that are a leaf in the category tree,
```

Comment:
Same as in the demo site, I don't really think this is the best idea

Reply:
see explanation above.